Dockerfile

```diff
@@ -1,31 +1,29 @@
-
 FROM osgeo/gdal:ubuntu-small-3.3.1
 
 ENV DEBIAN_FRONTEND=noninteractive \
     LC_ALL=C.UTF-8 \
-    LANG=C.UTF-8
+    LANG=C.UTF-8 \
+    CIABPASSWORD=secretpassword
 
 RUN apt-get update && \
     apt-get install -y \
       build-essential \
-      fish \
       git \
       # For Psycopg2
-      libpq-dev python-dev \
+      libpq-dev python3-dev \
       python3-pip \
       tini \
       wget \
-    && apt-get autoclean && \
-    apt-get autoremove && \
-    rm -rf /var/lib/{apt,dpkg,cache,log}
+    && apt-get autoclean \
+    && apt-get autoremove \
+    && rm -rf /var/lib/{apt,dpkg,cache,log}
 
-RUN mkdir -p /conf
 COPY requirements.txt /conf/
-RUN pip install -r /conf/requirements.txt \
-    && rm -rf /var/lib/apt/lists/*
+RUN pip3 install --requirement /conf/requirements.txt \
+      && rm -rf /root/.cache
 
 WORKDIR /notebooks
 
 ENTRYPOINT ["/bin/tini", "--"]
```

Comment:
I actually don't think tini installs properly this way.

In another image, I did this instead:

```
# Add Tini
ENV TINI_VERSION v0.19.0
ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
RUN chmod +x /tini
ENTRYPOINT ["/tini", "--"]
```